packages/shared-models/package.json

```diff
@@ -37,7 +37,7 @@
     "watch": "tsc -b --watch"
   },
   "dependencies": {
-    "@jupyterlab/nbformat": "^4.0.0-alpha.11",
+    "@lumino/algorithm": "^1.9.1",
```

Comment:
Maybe we should call that package `shared-types`.

Reply:
I think this proposed `@jupyterlab/shared-types` package should export a single namespace called `SharedTypes` with both interfaces and concrete implementations, e.g., `SharedTypes.List` and `SharedTypes.IList`. This will minimize the number of individually named items a consumer of the library will have to `import` to use it.